kafkacat/README.adoc

```diff
@@ -91,9 +80,14 @@ Copyright (c) 2014-2019, Magnus Edenhill
 Version 1.6.0 (JSON, Avro, Transactions, librdkafka 1.6.1 builtin.features=gzip,snappy,ssl,sasl,regex,lz4,sasl_gssapi,sasl_plain,sasl_scram,plugins,zstd,sasl_oauthbearer)
 ----
```

Comment:
@PaulRMellor Same suggestion as before with scripts, maybe streamline the prereq as follows:

```
.Prerequisites
ifndef::community[]
* You have a Red Hat account.
endif::[]
//* You have a subscription to {product-long}. For more information about signing up, see *<@SME: Where to link?>*.
* You have a running Kafka instance in {product}.
* You have installed the latest version of https://github.com/edenhill/kafkacat[Kafkacat^] for your operating system and verified the installation.
+
.Verifying Kafkacat installation
[source]
----
$ kafkacat -V

kafkacat - Apache Kafka producer and consumer tool
https://github.com/edenhill/kafkacat
Copyright (c) 2014-2019, Magnus Edenhill
Version 1.6.0 (JSON, Avro, Transactions, librdkafka 1.6.1 builtin.features=gzip,snappy,ssl,sasl,regex,lz4,sasl_gssapi,sasl_plain,sasl_scram,plugins,zstd,sasl_oauthbearer)
----
* https://adoptopenjdk.net/[JDK^] 11 or later is installed.
* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
```

Or this:

```
.Prerequisites
ifndef::community[]
* You have a Red Hat account.
endif::[]
//* You have a subscription to {product-long}. For more information about signing up, see *<@SME: Where to link?>*.
* You have a running Kafka instance in {product}.
* https://adoptopenjdk.net/[JDK^] 11 or later is installed.
* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
* You have installed the latest version of https://github.com/edenhill/kafkacat[Kafkacat^] for your operating system and verified the installation.
+
.Verifying Kafkacat installation
[source]
----
$ kafkacat -V

kafkacat - Apache Kafka producer and consumer tool
https://github.com/edenhill/kafkacat
Copyright (c) 2014-2019, Magnus Edenhill
Version 1.6.0 (JSON, Avro, Transactions, librdkafka 1.6.1 builtin.features=gzip,snappy,ssl,sasl,regex,lz4,sasl_gssapi,sasl_plain,sasl_scram,plugins,zstd,sasl_oauthbearer)
----
```

Reply:
Done